src/shipment/controller.js

```diff
@@ -505,3 +510,323 @@ async function getPopulatedShipmentData(shipmentId = undefined) {
     return [e];
   }
 }
+
+async function getAsPdf(req, res) {
+  ExpressHandler(
+    async () => {
+      const { manifest, customer, dateCreated, shipmentId, deliveryMethod } = req.body;
+      const { title } = customer;
+
+      const orderNumbers = [];
+      for ( m of manifest ) {
+        if ( !orderNumbers.includes(m.orderNumber) ) orderNumbers.push(m.orderNumber);
```

Comment:
This is a tad hard to read.. the same thing might be accomplished with a `Set`
for example `const orderNumbers = new Set( manifest.map(x => x.orderNumber) )`  and you can still iterate on orderNumbers as usual

Reply:
Updated `orderNumbers` to be a Set. Had to add a counter ( `idx` ) to determine if `pageBreakAfter` is set to true or false.